week-2/B-boolean-literals/exercise.js

```diff
@@ -16,6 +16,12 @@ console.log("Is Mozafar cool?", mozafarIsCool);
 console.log("Does 1 + 1 = 2?", calculationCorrect);
 console.log("Are there more than 10 students?", moreThan10Students);
 
+module.exports = {
+  mozafarIsCool,
+  calculationCorrect,
+  moreThan10Students
+};
```

Comment:
Could we move the module.exports at the end of the file with a message like "IGNORE BELOW" please? 
I want it to be as hidden as possible, so it doesn't confuse them :D